packages/block-library/src/button/editor.scss

```diff
@@ -1,9 +1,3 @@
-.wp-block[data-align="center"] > .wp-block-button {
```

Comment:
I think this rule might still be required for the legacy standalone Button block if it appears outside of a Buttons container.

To get the legacy button block in there, If you add a paragraph to your post, then switch to the code editor view, and insert the following (a single Button block) at the end of the post content, then it'll expose the alignment controls for the standalone button:

```
<!-- wp:button -->
<div class="wp-block-button"><a class="wp-block-button__link">Button A</a></div>
<!-- /wp:button -->
```

The left and right alignment appears to be working the same as on trunk, however the center alignment in the editor looks like this if we remove this CSS rule:

![image](https://user-images.githubusercontent.com/14988353/132635995-c216e05b-30ea-4b29-a48f-ea1a047901c0.png)

Reply:
Yep, missed this one. Thanks!